app/scripts/lib/createDupeReqFilterMiddleware.test.js

```diff
@@ -9,8 +9,8 @@ describe('createDupeReqFilterMiddleware', () => {
 
```

Comment:
@darkwing did this file need to change?

Reply:
*sigh* No, `yarn lint:fix` was correcting it and I needed to keep restoring :/